### ISSUE TITLE
reset(): Argument #1 ($array) must be passed by reference, value given

### ISSUE THREAD

**Issue by opsquid** [haruncpi/laravel-user-activity#17]
hi,

I'm getting this error when trying to access /admin/user-activity:
reset(): Argument #1 ($array) must be passed by reference, value given

I'm using laravel 8 and php 8.

![image](https://user-images.githubusercontent.com/542911/105639761-990bb800-5eac-11eb-8629-7552795128b7.png)

**Comment by samehdoush** [comment]
I have the same problem 


edit  ActivityController
to : 
          ```
       switch ($driver) {
                        case 'pgsql':
                            $sql = sprintf(
                                "SELECT table_name FROM information_schema.tables where table_schema = '%s' ORDER BY 
                   table_schema,table_name;",
                                DB::connection($connection)->getConfig('schema') ?: 'public'
                            );
                            $all = array_map('reset', DB::select($sql));
                            break;
                        default:
                        $sql = sprintf(
                              "SELECT table_name FROM information_schema.tables where table_schema = '%s' ORDER BY table_schema,table_name;",
                            DB::connection($connection)->getConfig('schema') ?: 'public'
                        );
                            $all = array_map('reset', DB::select($sql));
```

**Comment by haruncpi** [comment]
`reset(): Argument #1 ($array) must be passed by reference, value given` fixed in v1.0.3
Please update to v1.0.3. Hope it'll work with php 8.0
